third_party/airflow/armada/operators/armada.py

```diff
@@ -77,57 +70,79 @@ def __init__(
         self.job_request_items = job_request_items
         self.lookout_url_template = lookout_url_template
         self.poll_interval = poll_interval
+        self.job_id = None  # Initialize job_id attribute
+        self.job_set_id = None  # Initialize job_set_id attribute
```

Comment:
You should add queue here too.

Reply:
@kannon92  sorry I forget to generate new pr for docs

Reply:
@kannon92 pls review the changes!!